Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php

```diff
@@ -0,0 +1,529 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+class MethodSeparationFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideFixClassesCases
+     */
+    public function testFixClasses($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideFixClassesCases()
+    {
+        $cases = array();
+        $cases[] = array(
+            '<?php
+abstract class MethodTest2
+{
+    public function method045()
+    {
+        $files = null;
+            if (!empty($files)) {
+            $this->filter(
+                function (\SplFileInfo $file) use ($files) {
+                    return !in_array($file->getRelativePathname(), $files, true);
+                }
+            );
+        }
+    }
+
+     private $a;
+
+     public static function method145()
+     {
+     }
+
+       abstract protected function method245();
+
+    // comment
+
+    final private function method345()
+    {
+    }
+}
+function test1(){ echo 1;}
+function test1(){ echo 2;}',
+            '<?php
+abstract class MethodTest2
+{
+    public function method045()
+    {
+        $files = null;
+            if (!empty($files)) {
+            $this->filter(
+                function (\SplFileInfo $file) use ($files) {
+                    return !in_array($file->getRelativePathname(), $files, true);
+                }
+            );
+        }
+    }
+     private $a;
+
+     public static function method145()
+     {
+     }
+       abstract protected function method245();
+    // comment
+
+    final private function method345()
+    {
+    }
+}
+function test1(){ echo 1;}
+function test1(){ echo 2;}',
+        );
+
+        // spaces missing below methods
```

Comment:
I get it, the comment was a left over TODO marker for my self to fix such case, this is already done so I've removed the comment https://github.com/SpacePossum/PHP-CS-Fixer/commit/0e3b1ad66c4c171f5e27ed6e0582bae206d3e3e1